README.md

```diff
@@ -18,7 +18,7 @@ You include Delta Lake in your Maven project by adding it as a dependency in you
 <dependency>
```

Comment:
Can you also update the text above to the remove the two sentences 

```Delta Lake is cross compiled with Scala versions 2.11 and 2.12; choose the version that matches your project. If you are writing a Java project, you can use either version.```

This is because its only scala2.12 now.

Reply:
Done 👍